core/jvm/src/main/scala/scalaz/zio/blocking/Blocking.scala

```diff
@@ -52,7 +54,7 @@ object Blocking extends Serializable {
      * If the returned `IO` is interrupted, the blocked thread running the synchronous effect
      * will be interrupted via `Thread.interrupt`.
      */
-    def interruptible[A](effect: => A): ZIO[R, Throwable, A] =
+    def interruptible[A](effect: => A): ZIO[R with Clock, Throwable, A] =
```

Comment:
This should not use `Clock` IMO, but rather low-level facilities. It's already slow as it is, and if we pull in all this machinery, it will be slower. Also even in a test, to really interrupt blocking IO, we might need "real time" not "fake time" and using Clock introduces the possibility fo fake time.